server/db/models/order.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Order = db.define('Order', {
```

Comment:
I think we would want to make the "O" of the 'Order' a small case, just following the patterns from other models!